doc/Language/glossary.pod6

```diff
@@ -167,8 +167,8 @@ sub the-answer() { 42 }                            # arity of 0
 The arity of a C<Callable> is one of the main selectors in
 L<multi-dispatch|#Multi-dispatch>.
 
-=head1 ASCII operator
 X<|ASCII operator>
+=head1 ASCII operator
 X<|Texas operator>
 =begin comment
```

Comment:
It's probably sensible to just eliminate that index, together with the comment.

Reply:
Do you mean `Texas operator`?